ACE/ace/config-sunos5.10.h

```diff
@@ -59,4 +59,9 @@
 
 #define ACE_HAS_SOLARIS_ATOMIC_LIB
 
+// Solaris Studio 12.4 implements symbol lookup correctly.
+#if defined (__SUNPRO_CC) && (__SUNPRO_CC >= 0x5130)
+#define ACE_ANY_OPS_USE_NAMESPACE
+#endif
```

Comment:
a legit question... I pondered it for a while, but then I noticed, that most of the defines currently in config-suncc-common.h were already in some of the sunos* headers.    Getting each define into where it really belongs may be another year's work ;-)